docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -8,101 +8,126 @@ ms.custom: "seodec18"
 ---
 # Unit testing C# in .NET Core using dotnet test and xUnit
 
-This tutorial takes you through an interactive experience building a sample solution step-by-step to learn unit testing concepts. If you prefer to follow the tutorial using a pre-built solution, [view or download the sample code](https://github.com/dotnet/samples/tree/master/core/getting-started/unit-testing-using-dotnet-test/) before you begin. For download instructions, see [Samples and Tutorials](../../samples-and-tutorials/index.md#viewing-and-downloading-samples).
+This tutorial shows how to build a solution containing a unit test project and source code project. To follow the tutorial using a pre-built solution, [view or download the sample code](https://github.com/dotnet/samples/tree/master/core/getting-started/unit-testing-using-dotnet-test/). For download instructions, see [Samples and Tutorials](../../samples-and-tutorials/index.md#viewing-and-downloading-samples).
 
-[!INCLUDE [testing an ASP.NET Core project from .NET Core](../../../includes/core-testing-note-aspnet.md)]
+## Create the solution
 
-## Creating the source project
-
-Open a shell window. Create a directory called *unit-testing-using-dotnet-test* to hold the solution.
-Inside this new directory, run [`dotnet new sln`](../tools/dotnet-new.md) to create a new solution. Having a solution 
-makes it easier to manage both the class library and the unit test project.
-Inside the solution directory, create a *PrimeService* directory. The directory and file structure thus far should be as follows:
+In this section, a solution is created that contains the source and test projects. The completed solution has the following directory structure:
 
 ```
 /unit-testing-using-dotnet-test
     unit-testing-using-dotnet-test.sln
     /PrimeService
+        PrimeService.cs
+        PrimeService.csproj
+    /PrimeService.Tests
+        PrimeService_IsPrimeShould.cs
+        PrimeServiceTests.csproj
 ```
```

Comment:
This could be confusing to beginners. The full folder structure (tree) should come after the instructions that creates the solution and the projects with all this files.

Reply:
I disagree. As they go on they can refer to that.